### ISSUE TITLE
How to embed in Markdown&html outside Rmd and shiny

### ISSUE THREAD

**Issue by huyingjie** [davidgohel/ggiraph#84]
My blog uses hexo, which is a static website generator. Each blog post is a Markdown file. `blogdown` developed by Yihui Xie has a lot of errors and I do not have the knowledge to fix them. I do not use shiny. 

How to embed the animation plot generated in a Markdown file? Do you have javascript link similar to bootstrap`<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/js/bootstrap.min.js"></script>`? 

How to get the javascript code containing data which generates the animation plot?

Thank you.

**Comment by davidgohel** [comment]
Hi, sorry, closing that one (not an issue, cross posted in many repo and on SO).

**Comment by caitlinhudon** [comment]
Would you mind posting an example of the cross-post? I have the same question and am searching other closed issues but having trouble finding the answer. This might help others looking for the answer in the future.

**Comment by huyingjie** [comment]
@caitlinhudon 
My solution to Hexo.

1. Generate a html file which contains an animation from RStudio
2. In the Hexo `_config.yml`, set `post_asset_folder` to `true`.
    `post_asset_folder: true`
3. Add following code to `_config.yml`
	```
	skip_render:
	    - _posts/**/**.js
	    - _posts/**/**.html
	```
3. Create a Markdown blog post
4. Create a foler in the same path and its name is the same with the Markdown file.
5. Copy the HTML file to the folder 
2. In the MarkDown file, write `iframe` HTML elemnts to embed the HTML file, such as, `<iframe width=100% height=400 src="htmlFileName.html" frameborder="0" align="middle"></iframe>`

You need to adjust values of `height` and `width` in `iframe`to fit the graph in the HTML file embeded.

**Comment by davidgohel** [comment]
Hi,
 
No sorry. I will not list all identical issues @huyingjie posted as it would link them into that repository. For the one on SO, I guess the question has been downvoted and deleted. For the others, google the title of the *issue*.

> blogdown developed by Yihui Xie has a lot of errors and I do not have the knowledge to fix them.

It's not OK to open the same issue in (>5) repositories and say project X is rubbish and bring no evidence. In my experience, blogdown is working very well. 

> How to embed the animation plot generated in a Markdown file? [...]

As an `IFRAME`.

> How to get the javascript code containing data which generates the animation plot?

In modern navigator, you can access all these elements in *developer view*.

**Comment by huyingjie** [comment]
@davidgohel 
You replied me `Hi, sorry, closing that one (not an issue, cross posted in many repo and on SO).`
Obviously, you did not give me a solution to my question in your first answer. Even if I post on over 5 repos, your reply was rude.

`iframe` is not sufficient to solve the question. If `iframe` is the effective answer, I wouldn't open issues on at least 5 repos. 

I opened at least 5 issues on GitHub and none of the authors of repos gave me workable answers. I found the solution by myself.

**Comment by huyingjie** [comment]
When I asked the question in the past, I did not care about blogdown anymore. Mentioning `blogdown` was to show that I already did a lot of work searching online to look for solutions but failed. 

I had thought about the question for several days before asking. The simple and should-be solution is to provide related javascript file(s), and the html file which contains the graph only and without any styling (because I want to use my blog's styling, not rmd style and it is hard to delete all styling from html files generated by rmd. It is time-consuming when considering drawing a lot of graphs in the future) directly from blogdown package. But I do not think the author of blogdown will be willing to update the package to do it. In addition, I did not have the ability to combine the knowledge of hexo, JavaScript, jQuery, R, blogdown package, and JavaScript library of dynamic graphs together to write my own package.

I asked the question in different repos of R packages which created dynamic graphs. I hadn't decided which R package to use at that time.  I thought I would use whichever package whose owner can give me the solution to embed the graphs without creating rmd file. 

The result is that only one author kindly replied me but it did not solve the problem.  Your first reply definitely cannot be counted as a reply to the solution.
